src/js/popup.js

```diff
@@ -637,9 +637,9 @@ Logic.registerPanel(P_CONTAINERS_LIST, {
 
       context.classList.add("userContext-wrapper", "open-newtab", "clickable");
       manage.classList.add("show-tabs", "pop-button");
-      manage.title = escaped`View ${identity.name} container`;
+      manage.setAttribute("title", `View ${identity.name} container`);
       context.setAttribute("tabindex", "0");
-      context.title = escaped`Create ${identity.name} tab`;
+      context.setAttribute("title", `Create ${identity.name} tab`);
```

Comment:
Nice. This still avoids the self-XSS potential too.
![image](https://user-images.githubusercontent.com/71928/66935473-a1b2e480-f001-11e9-95ff-128b24696cd1.png)